### ISSUE TITLE
"Failed to load game data from Steam. Are you logged on the Steam Store?" 

### ISSUE THREAD

**Issue by saladofstones** [SteamDatabase/steamdb.info#645]
I left a previous topic regarding this issue, but it was closed despite it still occurring.

I have third party cookies enabled and this issue is still occurring. I was told that there was no one solution, but could I have some of them to debug this?

**Comment by xPaw** [comment]
Making duplicate issues is not useful to anyone.

You can try looking at the background page of extension and seeing what requests it does and what the answer is.

![04-2133-HelplessThunderbird](https://user-images.githubusercontent.com/613331/70183429-750c7680-16ee-11ea-8711-dfcbab798d90.png)

![04-2134-DarkvioletEastsiberianlaika](https://user-images.githubusercontent.com/613331/70183496-92d9db80-16ee-11ea-9e69-d9c4cda85ee7.png)

**Comment by saladofstones** [comment]
I couldn't see how to get into the background page, but I did try accessing the developer tools.

I use waterfox, and it displays that it synced correctly.

https://i.imgur.com/YHZvLRl.png

"Content Security Policy: Ignoring “https://steamdb.info” within script-src: ‘strict-dynamic’ specified  (unknown)
Content Security Policy: Ignoring “https://www.google-analytics.com” within script-src: ‘strict-dynamic’ specified  (unknown)
[bugsnag] Loaded!  nihilanth.5b2c17a705cb98b162c49e434868caa4.js:1:40695
Loading failed for the <script> with source “https://www.google-analytics.com/analytics.js”.  user:342
[SteamDB] Failed to load userdata Are you logged on the Steam Store?  common.js:73:72
[SteamDB] Userdata loaded Packages: 10987"

I am sorry if this isn't helpful, but this is what I could bring up for errors.

**Comment by saladofstones** [comment]
I installed chrome and the extension there worked without any issues. I think the firefox version is just broken.

**Comment by xPaw** [comment]
Open `about:debugging#/runtime/this-firefox` and inspect the extension.

I just tested it in Firefox, it works fine and loads data for me.

Try disabling other extensions you have installed.

**Comment by saladofstones** [comment]
yeah I didn't see anything about the cookies being blocked. I use waterfox, so it may be incompatible.

**Comment by sinkillerj** [comment]
I'm also having this issue with Waterfox.

**Comment by xPaw** [comment]
If it works in Firefox but not in Waterfox, report it to them. We don't support that browser.

**Comment by oprypin** [comment]
I was getting this problem on Firefox. The cause was [Multi-Account Containers](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/). 

I inspected in [`about:debugging#/runtime/this-firefox`](about:debugging#/runtime/this-firefox), thanks for the pointer.

The extension makes this kind of request and the result is empty.
https://store.steampowered.com/dynamicstore/userdata/?_=<\d+>
When I put the same URL in the browser manually, the response was properly populated.

I put Steam and SteamDB in the same container and logged in only there. The extension makes the request outside of any container, and I am indeed not logged in there.

The only workaround I found was to stop using containers.
